frontend/packages/console-app/src/components/guided-tours/DrawerComponent.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerPanelBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  Title,
+} from '@patternfly/react-core';
+import { GuidedTourItem, GuidedTourState } from './utils/guided-tour-typings';
+import './GuidedTourDrawer.scss';
+
+type DrawerComponentProps = {
+  guidedTour: GuidedTourItem;
+  guidedTourState?: GuidedTourState;
+  expanded: boolean;
+  onClose: () => void;
+  children?: React.ReactNode;
+  additionalHeaderContent?: React.ReactElement;
+};
+
+const DrawerComponent: React.FC<DrawerComponentProps> = ({
+  expanded,
+  guidedTour,
+  onClose,
+  children,
+  additionalHeaderContent,
+}) => {
+  const { name, duration } = guidedTour ?? {};
+  const panelContent = (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <div>
+          <Title headingLevel="h1" size="xl" className="co-guided-tour-drawer__title">
+            {name}
+          </Title>
+          <Title headingLevel="h6" size="md" className="co-guided-tour-drawer__subtitle">
```

Comment:
Use `text-secondary` which is mapped to `color: $color-text-secondary`. Or use the variable itself.

```suggestion
          <Title headingLevel="h6" size="md" className="co-guided-tour-drawer__subtitle text-secondary">
```

Reply:
done